core/src/mindustry/world/blocks/defense/turrets/Turret.java

```diff
@@ -425,12 +426,12 @@ protected void shoot(BulletType type){
                     float i = (shotCounter % shots) - (shots-1)/2f;
 
                     tr.trns(rotation - 90, spread * i + Mathf.range(xRand), shootLength);
-                    bullet(type, rotation + Mathf.range(inaccuracy));
+                    bullet(type, rotation + Mathf.range(inaccuracy + type.inaccuracy));
                 }else{
                     tr.trns(rotation, shootLength, Mathf.range(xRand));
 
                     for(int i = 0; i < shots; i++){
-                        bullet(type, rotation + Mathf.range(inaccuracy + type.inaccuracy) + (i - (int)(shots / 2f)) * spread);
+                        bullet(type, rotation + Mathf.range(inaccuracy + type.inaccuracy) + (i - shots / 2f) * spread);
```

Comment:
Why did you remove the int cast? This breaks spread.

![image](https://user-images.githubusercontent.com/10100000/126397248-db81f312-cda1-4a88-b0b3-f9260f128840.png)